index.html

```diff
@@ -52,6 +58,7 @@ <h3 class="socials-header">LET'S CONNECT</h3>
                 </li>
             </ul>
            </div>
+            </div>
         </section>
         <section class="works-sec">
             <div class="work-card">
```

Comment:
- [ ] The images in each card are not properly centered. Kindly fix it, as the previous reviewer recommended.

### Your design
![image](https://user-images.githubusercontent.com/86200356/206877311-d6d52643-c186-469f-a212-4e13541533a2.png)

### Figma
![image](https://user-images.githubusercontent.com/86200356/206877326-13688538-1415-45ff-97a6-567416382270.png)

Reply:
@Olamarx 
I'm honestly confused now
![Screenshot from 2022-12-11 01-30-22](https://user-images.githubusercontent.com/58359456/206877767-225ccc53-9bc1-4034-a9a3-104382aa833d.png)
This is what I see on my browser.

Reply:
No hamburgers no overflow. :weary:

Reply:
@Zeph180 I am in your dm on slack, do you have time for zoom call??